cosmos/launchpad/util.go

```diff
@@ -110,3 +114,17 @@ func toOperations(msg []cosmosclient.Msg, hasError bool) (operations []*types.Op
 	}
 	return
 }
+
+func getFromAddressFromOperations(ops []*types.Operation) (sdk.Address, error) {
+	var (
+		fromAddress sdk.Address
+		err         error
+	)
+	for _, op := range ops {
+		if strings.HasPrefix(op.Amount.Value, "-") {
```

Comment:
Not blocking, `isSender := strings.HasPrefix(op.Amount.Value, "-")` may help for future readability.

Reply:
I'll leave this to @jgimeno as it looks like this function will be replaced with his from payloads